bin/crew

```diff
@@ -202,10 +203,9 @@ def generate_compatible
   puts 'Generating compatible packages done.'.orange if @opt_verbose
 end
 
-def search(pkg_name, silent = false)
-  pkg_path = File.join(CREW_PACKAGES_PATH, "#{pkg_name}.rb")
+def search(pkg_name, pkg_path: File.join(CREW_PACKAGES_PATH, "#{pkg_name}.rb"), silent: false)
```

Comment:
Why does this need to be a parameter?  Keep it simple!

Reply:
Please read the reasons I gave in the PR description...

Reply:
> I'm not sure why you think this wasn't working when I've been using it for over a month now without an issue.

Probably because it's been broken whenever I tried to do it.

> No, the whole purpose of adding this function was to remove the need for bash scripts and incorporate it as a crew command instead.

Again, I do not understand how you can be in a scenario where you don't know the names of the binaries you are uploading. I genuinely do not see a usecase for not passing a package name.

> Then add it back. We'll want to control verbosity.

Because we aren't using system commands anymore, we can't just print the `curl` command we are using. If it was possible to add this code back I would have.

> It was already written and simplifies the process for contributors. Because the code is complicated is not a valid reason for removing it altogether.
> This functionality was added for a reason

Again, what is the reason? When are you in a scenario where you aren't on the same architecture as the binaries you've built?

> Again, package name makes more sense than package path
> No need to do this at all.

Please see my other comment where I explained why this is not the case.

Reply:
Actually, I can probably re-implement some of the verbosity, I'll do that when I've got the time.

Reply:
Added back the verbosity that was practical to do.